libs/langchain/langchain/llms/bedrock.py

```diff
@@ -26,7 +27,21 @@ def prepare_input(
         cls, provider: str, prompt: str, model_kwargs: Dict[str, Any]
     ) -> Dict[str, Any]:
         input_body = {**model_kwargs}
-        if provider == "anthropic" or provider == "ai21":
+        if provider == "anthropic":
+            if prompt.startswith("Human:") and prompt.endswith("Assistant:"):
+                reformatted_prompt = prompt  # Already wrapped
+            else:
+                # Guard against common errors in specifying wrong number of newlines.
+                corrected_prompt, n_subs = re.subn(r"^\n*Human:", "Human:", prompt)
+                if n_subs == 1:
+                    reformatted_prompt = corrected_prompt
+
+                # As a last resort, wrap the prompt ourselves to emulate instruct-style.
+                else:
+                    reformatted_prompt = f"{'Human:'} {prompt}{'Assistant:'}"
```

Comment:
I'm a bit new to the Langchain codebase, but shouldn't this be
`reformatted_prompt = f"{'\n\nHuman:'} {prompt}{'\n\nAssistant:'}"`